src/datatypes/src/duration.rs

```diff
@@ -111,6 +111,24 @@ macro_rules! define_duration_with_unit {
                     val.0.value()
                 }
             }
+
+            impl TryFrom<Value> for Option<[<Duration $unit>]> {
```

Comment:
Do we still need these try from?

Reply:
> Do we still need these try from?

Not really, but since other variant seems also `impl TryFrom<Value>` so for the sake of completeness still keep them?